src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -184,7 +184,7 @@ class ConfigurationHelper implements Serializable {
 
     ConfigurationHelper withPropertyInValues(String key, Set values){
         withMandatoryProperty(key)
-        def value = config[key]
+        def value = config[key] instanceof GString ? config[key].toString() : config[key]
```

Comment:
`String value = config[key]`   

should be sufficient.

Reply:
Yes, that's true, if the possible values are only String, but then we could not check for another types like int, for example. But I just realize that there is no unit tests for this method, so I will add a couple of unit tests, thank you.

Reply:
oups, i wasn't aware of other types...